### DISCUSSION TITLE
How to render HTML tags of a text DB record?

### DISCUSSION THREAD

**Discussion by superbolt-x** [Budibase/budibase#4831] [Help] [answered]
Hello,

Am using a postgres table to store comments and in the text field I am replacing \n with <br> so then i can interpret that as a new line when showing the text comment - I checked handlebars docs and they suggest using the triple brackets but they are not working to me {{{ field }}}

Example: Hello\nWorld -> loaded to DB as Hello[HTML TAG BREAK]World
Expected result with  {{{ }}} : 
Hello
Wolrd

Instead : Hello[HTML TAG BREAK]World

Any thoughts on that?

Thanks!!

**Comment by mjashanks** [comment, marked as answer]
Hey @superbolt-x 

Firstly, I just tested how Budibase handles `/n`, when they come from postgres. I found that it correctly puts a line break in, using both a `Paragraph` component and `Long form text` component (which also handles markdown btw). So, you might not need to use HTML tags at all.

However, I know that this is not what you asked! 

I'm not sure why  `{{{ HTML_tags }}}` doesn't work. However, you could use an `Embed` component - which does exactly what you are asking for - it takes HTML and puts it in the DOM directly.

**Reply by superbolt-x** [reply]
Thanks for the answer!!

**Reply by CovantaDataManagement** [reply]
I am trying to figure out how to close the current app tab using an HTML tag something like this:
<a href="javascript:closeWindow();">close</a>
I know there is an X but the user is rigid on this.  How might I do this?

**Reply by shogunpurple** [reply]
Hey @CovantaDataManagement - this will work for you:

- Create an embed component
- Add the following code
```
<button onclick="window.close()">
Close
</button>
```

Thanks!
